src/components/OneMonsterItem.js

```diff
@@ -66,9 +81,12 @@ const ParagraphCR = styled.p`
 const DetailLink = styled.img`
   max-height: 50px;
   max-width: 50px;
-  border: 2px solid var(--highlightcolor);
   border-radius: 6px;
+  /* border-width: 2px;
+  border-style: solid; */
+  /* border-color: ${({ monstertype }) => typeColors[monstertype]}; */
   padding: 10px;
   margin: 5px 10px;
   color: white;
```

Comment:
Maybe you can use also an var const fot the white color

Reply:
Yes good spot: Ill ninclude it in my ToDo list for my next CodeCleanup